src/chrome/content/rules/OpenX.net.xml

```diff
@@ -18,29 +18,22 @@
 -->
 <ruleset name="OpenX.net (partial)">
```

Comment:
We had a discussion about this somewhere, but I cannot find that issue anymore.
When major parts of the site cannot be secured, or only the resources that are fetched in background (so the https:// is not visible to the user), it makes sence to have the `(partial)` flag.
In this case the `www` host has no content, so it's trivial. In other cases `www` is the most important host.